pyanaconda/installclass.py

```diff
@@ -64,7 +64,7 @@ class BaseInstallClass(object):
     default_autopart_type = DEFAULT_AUTOPART_TYPE
 
     # help
-    help_folder = "/usr/share/anaconda/help"
+    help_folder = "/usr/share/anaconda/help/rhel"
```

Comment:
Maybe this change should be in `RHELBaseInstallClass` instead of `BaseInstallClass`.